### ISSUE TITLE
wrong startTime in worklogs result

### ISSUE THREAD

**Issue by gbucher** [everit-org/jira-worklog-query-plugin#14]
we are currently experimenting with this plugin and found the following case, resulting in a wrong returing startDate of a worklog entry using /rest/jira-worklog-query/1.2.0/find/worklogs

compare timestamp of worklog 23634, timestamp should be 2015-03-19T12:57:00+0100

``` json
{
  "id": 23634,
  "startDate": "2015-03-19T00:57:00+0100",
  "issueKey": "BIT-3",
  "userId": "<user>",
  "duration": 10800,
  "comment": "",
  "updated": "2015-03-19T00:59:34+0100"
},
{
  "id": 23635,
  "startDate": "2015-03-19T13:08:08+0100",
  "issueKey": "BIT-3",
  "userId": "<user>",
  "duration": 300,
  "comment": "more time spent",
  "updated": "2015-03-19T13:08:08+0100"
},
```

compare to jira's rest api result https://<server>/rest/api/2/issue/BIT-3/worklog/23634

``` json
{
    "self": "https://<server>/rest/api/2/issue/80923/worklog/23634",
    ....
    "comment": "",
    "created": "2015-03-19T12:58:07.094+0100",
    "updated": "2015-03-19T12:58:07.094+0100",
    "started": "2015-03-19T12:57:00.000+0100",
    "timeSpent": "3h",
    "timeSpentSeconds": 10800,
    "id": "23634"
}
```

I noticed, it happens only between 12:00 and 12:59, startDates before 11 or after 13 (see worklog 23635) are correct. any idea?

**Comment by balazs-zsoldos** [comment]
Thanks for the report. We will have a look.

**Comment by gbucher** [comment]
thanks... just tested your fix and all fine.

**Comment by balazs-zsoldos** [comment]
Fixed in 1.2.1
